app/models.py

```diff
@@ -14,6 +14,7 @@ class Team(db.Model):
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     name = db.Column(db.String(), unique=True, nullable=False)
     creator_id = db.Column(db.Integer, db.ForeignKey("User.id"))
+    active = db.Column(db.Boolean, nullable=False, default=True)
```

Comment:
```suggestion
    active = db.Column(db.Boolean, nullable=False, default=True, server_default=text("true"))
```
So ist der server_default auch im model hinterlegt